### DISCUSSION TITLE
点歌 Batch Normalization

### DISCUSSION THREAD

**Discussion by yqy2001** [mli/paper-reading#57] [General]
深度学习领域引用量最高的经典paper之一，几乎所有神经网络都会用到的层～

**Comment by fengshi-cherish** [comment]
虽然在transosformer中已经稍微讲过bn和ln了，但还是希望能更细节的讨论关于norm的部分。

**Comment by Jazihars** [comment]
或许可以参考李老师的[动手学深度学习BN课程](https://www.bilibili.com/video/BV1X44y1r77r?spm_id_from=333.999.0.0)？

**Comment by mli** [comment]
这个在b站上放的两门课就见过，论文就不准备再读了
